docs/plaindate.md

```diff
@@ -278,9 +280,10 @@ Usage example:
 
 ```javascript
 // Attempt to write some mnemonic poetry
+({ calendar } = Intl.DateTimeFormat().resolvedOptions());
```

Comment:
This poem doesn't work in calendars outside the Gregorian family, so let's use ISO.